domain/src/main/java/org/oppia/domain/profile/ProfileTestHelper.kt

```diff
@@ -0,0 +1,33 @@
+package org.oppia.domain.profile
+
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.oppia.app.model.ProfileId
+import javax.inject.Inject
+
+class ProfileTestHelper @Inject constructor(
+  private val profileManagementController: ProfileManagementController
+) {
+  /** Creates one admin profile and one user profile. Logs in to admin profile. */
+  @ExperimentalCoroutinesApi
```

Comment:
Here & elsewhere in this helper: is this actually needed?

Reply:
This is needed because addProfile uses deferred.getCompleted() which needs to be marked by @ExperimentalCoroutinesApi.

Reply:
Nevermind, this is no longer needed after I changed ProfileManagementController.